CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 - Added devtool build `--ssh-keys` flag to support fetching from private
   git repositories.
 - Added option to configure block device flush.
+- Added '--jailer-time-cpu-us' flag to Firecracker, which represents the total
+  CPU time of the jailer process in us.
```

Comment:
Actually, these params (`start_time_X`) were only supposed to be used by the CI for latency regression testing. They were originally hidden and did not even show up in `--help` - since moving from `clap` to `arg_parser` we lost the _hidden_ property and they are now visible which is a different issue altogether.

Long term we should aim to deprecate and remove these params and move to a different measuring mechanism in CI (use `perf` for example). The current method of firecracker measuring itself while taking into account parents' time, then expose that through its metrics is weird and as can be seen here, does not scale/is not flexible with other usecases. For now, we're stuck with adding yet another param for these measurements 😢 

Since this (`--jailer-time-cpu-us`) is supposed to only be used internally, let's not add it to the CHANGELOG, it's only distracting without being useful.

Reply:
Opened issue here: https://github.com/firecracker-microvm/firecracker/issues/2516